backend/index.js

```diff
@@ -70,6 +70,116 @@ app.get('/', (req, res) => {
     res.send('game selection');
 });
 
+// devtools
+app.get('/devtools', (req, res) => {
+    res.send('developer tools');
+});
+
+app.get('/words', async (req, res) => {
+    try {
+        const wordsData = await prisma.words.findFirst();
+        if (!wordsData) {
+            wordsData = await prisma.words.create({
```

Comment:
btw, in production-scale systems, this pattern introduces a race condition. Two different web servers may be serving the same request, and will race to insert `Words`.

you can solve this in a few ways, but I think it's moot because we want to move to a different `Words` table structure; I will leave it as extra credit at your leisure to find what solutions look like :)